src/client/main.c

```diff
@@ -0,0 +1,60 @@
+#include <getopt.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "client.h"
+#include "libhirte/common/common.h"
+#include "libhirte/common/opt.h"
+
+const struct option options[] = { { ARG_HELP, no_argument, 0, ARG_HELP_SHORT }, { NULL, 0, 0, '\0' } };
+
+#define OPTIONS_STR ARG_HELP_SHORT_S
+
+// NOLINTBEGIN(cppcoreguidelines-avoid-non-const-global-variables)
+static char *op;
+static char **opargv;
+static int opargc;
+// NOLINTEND(cppcoreguidelines-avoid-non-const-global-variables)
+
+static void usage(char *argv[]) {
+        fprintf(stderr, "Usage: %s COMMAND ... \n", argv[0]);
+}
+
+static void get_opts(int argc, char *argv[]) {
```

Comment:
I think this function should return success or failure and allow main to do the exiting.

Reply:
Ack